components/Cards/index.js

```diff
@@ -17,3 +17,170 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+    .then((response) => {
+        const newCard = cardArticles(response.data.articles.bootstrap[0])
```

Comment:
You could also store the whole response into an array, for example=

const articles = Object.keys(response.data.articles);
Then you could use forEach on 'articles'